go.mod

```diff
@@ -15,9 +15,9 @@ require (
 	github.com/onsi/ginkgo/v2 v2.25.3
 	github.com/onsi/gomega v1.38.2
 	github.com/operator-framework/api v0.33.0
-	github.com/percona/percona-backup-mongodb v1.8.1-0.20241212160532-0157f87a7eee
+	github.com/percona/percona-backup-mongodb v1.8.1-0.20250327100109-3ca299246bfe
 	github.com/percona/percona-postgresql-operator v0.0.0-20250717192602-2b721be58505
-	github.com/percona/percona-server-mongodb-operator v1.19.1
+	github.com/percona/percona-server-mongodb-operator v1.20.1
```

Comment:
TODO: change to `1.21.0` once published. Need at least `1.20` to be able to set up the `main` attribute for the storage.